pkgs/desktops/xfce/panel-plugins/xfce4-wavelan-plugin/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchurl
+, pkg-config
+, intltool
+, xfce4-panel
+, libxfce4ui
+, xfconf
+, gitUpdater
+}:
+
+let
+  category = "panel-plugins";
+in stdenv.mkDerivation rec {
+  pname  = "xfce4-wavelan-plugin";
+  version = "0.6.3";
+
+  src = fetchurl {
+    url = "mirror://xfce/src/${category}/${pname}/${lib.versions.majorMinor version}/${pname}-${version}.tar.bz2";
+    sha256 = "sha256-YcDC9Wy3CHLUA7dw3XY0nfn/JMDb6QXuG0+RPDTY9ys=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    intltool
+  ];
+
+  buildInputs = [
+    libxfce4ui
+    xfce4-panel
+    xfconf
+  ];
+
+  passthru.updateScript = gitUpdater {
+    url = "https://gitlab.xfce.org/panel-plugins/${pname}";
+    rev-prefix = "${pname}-";
+  };
+
+  meta = with lib; {
+    homepage = "https://docs.xfce.org/panel-plugins/xfce4-wavelan-plugin";
+    description = "Display stats from a wireless LAN interface in the Xfce panel";
+    license = licenses.gpl2Plus;
```

Comment:
This is probably [not correct](https://gitlab.xfce.org/panel-plugins/xfce4-wavelan-plugin/-/blob/master/COPYING)?